sendKindle.py

```diff
@@ -91,7 +91,7 @@ def read_config(self):
         '''Parse the config file. Create a new one if needed.'''
 
         self.create_config()
-        config = configparser.SafeConfigParser()
+        config = configparser.ConfigParser()
```

Comment:
Why?

Reply:
I got a warning of "figParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead."

Reply:
OK, you should add this information to the commit message next time, and it's going to be clear.